File: sivemor-app/src/main/java/mx/edu/utez/sivemorapp/modules/evaluaciones/dtos/EvidenciaEvaluacionDTO.java
package mx.edu.utez.sivemorapp.modules.evaluaciones.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class EvidenciaEvaluacionDTO {
    private Long id;
    private Integer numeroEvidencia;
    private String nombreArchivo;
    private String mimeType;
    private Integer tamanoBytes;

    private String imagenBase64;
    private String comentario;
}

File: SivemorMobile/app/src/main/java/com/sivemor/mobile/network/dto/EvidenciaEvaluacionDTO.kt
package com.sivemor.mobile.network.dto

data class EvidenciaEvaluacionDTO(
    val numeroEvidencia: Int? = null,
    val nombreArchivo: String? = null,
    val mimeType: String? = null,
    val tamanoBytes: Int? = null,
    val comentario: String? = null,
    val imagenBase64: String? = null
)